src/act.obj.cpp

```diff
@@ -268,7 +268,12 @@ void perform_put_cyberdeck(struct char_data * ch, struct obj_data * obj,
 
     int space_required = 0;
     if (GET_OBJ_TYPE(obj) == ITEM_PROGRAM) {
-      space_required = GET_PROGRAM_SIZE(obj);
+      // Persona programs don't take up storage memory in store-bought decks
```

Comment:
Just to confirm, does this code change cover storage used when uninstalling as well?

Reply:
Oops, no it doesn't. And fixed below.